test/Microsoft.AspNet.SignalR.FunctionalTests/Client/WebSocketFacts.cs

```diff
@@ -33,7 +33,7 @@ public async Task ClientCanReceiveMessagesOver64KBViaWebSockets(HostType hostTyp
 
                     await connection.Start(host.Transport);
 
-                    var result = await hub.Invoke<string>("ReturnLargePayload").OrTimeout();
+                    var result = await hub.Invoke<string>("ReturnLargePayload").OrTimeout(TimeSpan.FromSeconds(30));
```

Comment:
When running in parallel, it can actually take longer than the default 5 second timeout for this to execute because the CPU is busy with a bunch of other tests. It's still faster overall to run in parallel though.